extensions/wikia/AdEngine/js/lookup/prebid/adaptersRegistry.js

```diff
@@ -81,8 +84,19 @@ define('ext.wikia.adEngine.lookup.prebid.adaptersRegistry', [
 		});
 	}
 
+	function getPriority() {
```

Comment:
We're returning a dictionary here. Can we make sure that method name suggests it? Eg. `getPriorities()`

Reply:
Sure, done.